app/main.py

```diff
@@ -1,3 +1,45 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: (int, float)) -> None:
```

Comment:
In future use Union[int, float] or int | float as @katevasilenko says 
In real projects, I never saw the usage of (int, float)